src/twisted/mail/pop3.py

```diff
@@ -855,7 +858,13 @@ def do_PASS(self, password):
 
         @type password: L{bytes}
         @param password: A password.
+
+        @type args: L{tuple} or L{None}
+        @param args: Other words composing the password.
         """
+        if args:
+            password += ' ' + ' '.join(args)
```

Comment:
Or `password = " ".join((password,) + args)`.

Regardless of that, `b" "` - not `' '`.  Byte strings need to be explicit now.